esp/templates/contact_qsd.html

```diff
@@ -0,0 +1,18 @@
+{% extends "main.html" %}
+
+{% block title %}Contact Us{% endblock %}
+{% block content_title %}Contact Us{% endblock %}
+
+{% block content %}
+
+{% load render_qsd %}
+
+{% inline_qsd_block "contact" %}
+
+<p>
+For general communication, please email us at <b><a href="mailto:{{ settings.DEFAULT_EMAIL_ADDRESSES.default }}">{{ settings.DEFAULT_EMAIL_ADDRESSES.default }}</a></b>.
```

Comment:
if the contact form is enabled, should we have a link to it on this default page??